magento/magento2
# Solution for other options values delete issue.

### Description

A solution for other options values deletes issue.

### Fixed Issues (if relevant)

1. magento/magento2#13425: Delete one value from Select type custom option and it's deleted all other custom option values which are bottom of that custom option.

### Manual testing scenarios

1.  Create a new product or edit existing product Add multiple Custom options having the type (Dropdown, multiple select, Radio Button, CheckBox)
2.  Save the Product
3. Open the product, delete one value from first select type custom option.
4. save the product
5. Check all other custom options values are not deleted which are bottom of that custom options.

### Contribution checklist
